LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-model/animation-types/addition-per-property-002-expected.txt

```diff
@@ -274,13 +274,13 @@ PASS word-break: "keep-all" onto "break-all"
 PASS word-spacing (type: lengthPercentageOrCalc) has testAddition function
 PASS word-spacing: length
 PASS word-spacing: length of rem
-FAIL word-spacing: percentage assert_equals: The value should be 130% at 0ms expected "130%" but got "3.25px"
-FAIL word-spacing: units "%" onto "px" assert_equals: The value should be calc(10% + 10px) at 0ms expected "calc(10% + 10px)" but got "0.25px"
-FAIL word-spacing: units "px" onto "%" assert_equals: The value should be calc(10% + 10px) at 0ms expected "calc(10% + 10px)" but got "3355443px"
-FAIL word-spacing: units "rem" onto "%" assert_equals: The value should be calc(10% + 20px) at 0ms expected "calc(10% + 20px)" but got "3355443px"
-FAIL word-spacing: units "%" onto "rem" assert_equals: The value should be calc(10% + 20px) at 0ms expected "calc(10% + 20px)" but got "0.25px"
+PASS word-spacing: percentage
+PASS word-spacing: units "%" onto "px"
+PASS word-spacing: units "px" onto "%"
+PASS word-spacing: units "rem" onto "%"
+PASS word-spacing: units "%" onto "rem"
 PASS word-spacing: units "rem" onto "em"
 PASS word-spacing: units "em" onto "rem"
 FAIL word-spacing: units "calc" onto "px" assert_equals: The value should be calc(20% + 30px) at 0ms expected "calc(20% + 30px)" but got "20px"
-FAIL word-spacing: calc assert_equals: The value should be calc(30% + 30px) at 0ms expected "calc(30% + 30px)" but got "0px"
+FAIL word-spacing: calc assert_equals: The value should be calc(30% + 30px) at 0ms expected "calc(30% + 30px)" but got "auto"
```

Comment:
I don't think `auto` is ever a valid value for `word-spacing`. The default value for word-spacing is `normal`.

Can you check why and fix up so we never return `auto`?

Reply:
The test failures on the GTK bot are a bit worrying: https://ews-build.webkit.org/#/builders/35/builds/32151